devel/CLI11/Portfile

```diff
@@ -1,24 +1,43 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
-PortGroup           github 1.0
 PortGroup           cmake 1.1
+PortGroup           github 1.0
 
-github.setup        CLIUtils CLI11 2.3.2 v
+github.setup        CLIUtils CLI11 2.4.0 v
 categories          devel
 license             BSD
 platforms           any
 supported_archs     noarch
-maintainers         {l2dy @l2dy} openmaintainer
-description         a command line parser for C++11
+maintainers         {l2dy @l2dy} {@barracuda156 gmail.com:vital.had} openmaintainer
+description         A command-line parser for C++11
 long_description    CLI11 is a command line parser for C++11 and beyond that \
                     provides a rich feature set with a simple and intuitive interface.
 
-checksums           rmd160  a94d978a883735a598ec122d85267cb32375ec6e \
-                    sha256  41dbf68beedf6e47355ef79e0c0debdf17b12859e170803d4cb772f1811dbbe1 \
-                    size    303644
+checksums           rmd160  b4a8acb42c8c5f1d28b83eaf013e6724d4014cac \
+                    sha256  d2ce8d5318d2a7a7d1120e2a18caac49cd65423d5d4158cbbc0267e6768af522 \
+                    size    338628
+github.tarball_from archive
+
+# https://github.com/CLIUtils/CLI11/commit/30ead5da4d00d10684930f900d3fcca8953d1cea
+patchfiles-append   0001-fix-transiative-include-issue-996.patch
```

Comment:
Why do we need this patch? Can we wait till the next release?

Reply:
Because it does not build otherwise. Or needs to set C++17 standard.

Reply:
Do you think it is preferable not to update the port at all until upstream decides to make a new release?

Well, it is your port and therefore your decision. If you insist, I could only try convincing upstream make a follow-up minor release.

Reply:
> Well, it is your port and therefore your decision. If you insist, I could only try convincing upstream make a follow-up minor release.

Please try that first. If upstream refuses, I will accept the current change as is.

Reply:
I have asked there, let’s see.

Reply:
@barracuda156 Could you try v2.4.1?